index.js

```diff
@@ -39,6 +39,13 @@ const webExtension = require("sdk/webextension");
 const windows = require("sdk/windows");
 const windowUtils = require("sdk/window/utils");
 
+const Metrics = require("./testpilot-metrics");
```

Comment:
I pulled this down below the others because I tend to do my imports in this structure:

1. Lowest-level/common-library modules
2. 3rd-party library modules
3. local modules
4. define local constants

But I'm happy to do alphabetical if that makes more sense.